package.json

```diff
@@ -15,5 +16,16 @@
   "bugs": {
     "url": "https://github.com/json-schema-org/json-schema-test-suite-npm/issues"
   },
-  "homepage": "https://github.com/json-schema-org/json-schema-test-suite-npm#readme"
+  "homepage": "https://github.com/json-schema-org/json-schema-test-suite-npm#readme",
+  "devDependencies": {
+    "ajv": "^6.10.0",
+    "assert": "^1.4.1",
+    "glob": "^7.1.3",
+    "json-schema-test": "^2.0.0",
+    "mocha": "^6.0.2",
+    "standard": "^12.0.1"
+  },
+  "dependencies": {
+    "json-schema-test-suite": "github:json-schema-org/JSON-Schema-Test-Suite#8982b0b"
```

Comment:
What is the difference between that package and this one?

Reply:
@jviotti When requiring a module as a git dependency, you need to use the same name as the one reported in it's `package.json`, hence it having the same name. This dependency is the original repo containing the test files, doing it this way allows us to stay in the npm/node universe to load the JSON test files and not do anything with git submodules etc

Reply:
Ah, I see